src/client/static/index.tsx

```diff
@@ -39,9 +37,11 @@ ophanInit();
 if (!window.Cypress) {
   // load cmp if it should show
   (async () => {
-    const country = await getCountryCode();
+    const country = await getLocale();
 
-    cmp.init({ country });
+    if (country) {
+      cmp.init({ country });
+    }
```

Comment:
What's the ux here if the call to getLocale fails? Do users then not get to set their consent prefs?

Just trying to work out if there's a default needed or if cmp really needs to know the country

Reply:
Note. This is a bit outside this PR, just curious really

Reply:
the cmp throws an error if the country is not supplied according to: https://github.com/guardian/consent-management-platform#optionscountry

i don't think we can set a default country/value

Reply:
yeah the users don't get to set their consent perfs, in turn meaning that we have to assume they havent consented to anything

Reply:
the only thing in gateway that requires consent is google analytics

Reply:
> we have to assume they havent consented to anything

Ah, this seems fair. I guess that's the best outcome 👍